_posts/2016-03-21-windows-support.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Windows support update
+categories: tech-issues future-plans
+---
+
+The OME team has always been committed to building specifications and software
+that are cross-platform solutions and that support as many different types of
+users as possible, including those with limited IT support/budget. In keeping
+with our open source ethos and the fact we rely on public and charity grant
+funding, we also try to use open source tools as part of our workflows for
+testing, building and deploying our products wherever possible.  We do support
+commercial OS and platforms — we build and test OMERO.insight on Windows,
+actively work to support accessing the OMERO and Bio-Formats APIs in Matlab,
+and actively support browsing OMERO.web using IE.
+
+Since the beginning of the OMERO project, we have actively tested and
+supported builds and tests of OMERO.server on Windows. Several users—
+students, faculty and institutions—have highlighted the importance of this
```

Comment:
Oh, I see you've used them elsewhere as brackets. So it's meant to be read like "Several users (students, faculty and institutions) have highlighted...". I've never seen dashes used like that before.